rpi-vi-first-steps.md

```diff
@@ -19,21 +19,29 @@ This equipment will be deployed in the field with little or no access to the int
 
 ### Pre-Requisites
 #### Hardware requirement:
-1. A Raspberry Pi 4 
-1. A Class 10 microSD card (minimal 16GB, but we recommend 32GB or greater)
-1. A microSD card reader (Check if your computer sd slot or if a usb/microSD card adapter is needed)
-1. 8 GB of harddrive space
-1. 8 GB of memory (RAM)
+1.  A Raspberry Pi 4
+	- Pi 3, Pi 3b+, Pi Zero W are also options
```

Comment:
We are encouraging them to go straight for RPi 4 so let's remove the others

Reply:
Done!